votes/0150-pr-issue-association.md

```diff
@@ -0,0 +1,51 @@
+# A Vote for PR Issue Association
+
+* Discussion at https://github.com/tikv/community/pull/150
+* Tracking issue https://github.com/tikv/community/issues/149
+
+## Proposal
+
+### Motivation
+
+To measure software evolution, we focus on features implemented, bugs fixed, and general enhancement. Issues document these items, while a pull request focuses on the change set and whether we implement it correctly.
+
+Normally, an issue documents a work item, and a pull request resolves it (despite cherry-picks). However, the relation between pull requests and issues is complicated in TiDB projects.
+
+* 1 issue with 0 PR: question, bug vanished without a reason, etc. 
+* 1 issue with 1 PR: normal case. 
+* 1 issue with n PRs: resolve a bug multiple times ([1](https://github.com/pingcap/tidb/issues/24679)), or resolve a complex issue in steps([2](https://github.com/pingcap/tidb/issues/27652)).
+* n issues with 1 PR: trade off resolving things as a whole (due to code coupling, or we can resolve multiple bugs at the common parent level) ([3](https://github.com/pingcap/tidb/pull/27697)), duplicated issues. 
+* 0 issue with 1 PR: coding without a ticket ([4](https://github.com/pingcap/tidb/pull/23022)). Even if the description is detailed or even if a design document is present, pull requests focus on the change set. It is about the implementation, not the question; if it is, we always mix discussion because the change set has a primary position. 
```

Comment:
(i'm not sure if it is a good idea to refer to use all the TiDB examples in the TiKV community 🙃)

Reply:
@kennytm Aha! Let me try to find other examples.

Reply:
Updated.